templates/paragraph/paragraph--localgov-banner-primary.html.twig

```diff
@@ -0,0 +1,74 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a paragraph.
```

Comment:
This is not actually the *default* template.  If you can please update the comment.